pkgs/term_glyph/tool/generate.dart

```diff
@@ -20,8 +20,10 @@ void main() {
   _writeGlyphSet(data, ascii: true);
   _writeTopLevel(data);
 
-  final result = Process.runSync(
-      'pub', ['run', 'dart_style:format', '-w', 'lib/src/generated']);
+  final result = Process.runSync('dart', [
```

Comment:
You could also do `Platform. resolvedExecutable` here (instead of `'dart'`).

Reply:
wise!